guardian/prosemirror-elements
# Bump `webpack-dev-server` to solve vulnerabilities

## What does this change?
This bumps webpack-dev-server to version `5`. It also bumps node to version `18` in order to accommodate that version bump. This should solve some high priority vulnerabilities associated with webpack-dev-server .

## How to test
Run `yarn` to install the dependencies 
Run `yarn start` to run the dev server. Does it run as expected?
